github_tutorial.md

```diff
@@ -0,0 +1,235 @@
+
+**upstream**
+Wanneer je op je eigen laptop/pc werkt aan bestanden, is "upstream" de alias voor de GitHub (remote) repository van VNG-Realisatie. Dit gebruik je wanneer je de lokale kopie van de repository wilt actualiseren met alle door anderen doorgevoerde wijzigingen (via een `git pull upstream master`).
+
+# GitHub workflow voor open Standaarden
+De workflow voor open co-creatie aan standaarden werkt met via "pull requests". Dit betekent dat je wijzigingen nooit direct maakt op de VNG Realisatie repository, zelfs niet direct op een branch in de VNG Realisatie repository.
+Je doet wijzigingen in je persoonlijke "fork" van de VNG Realisatie repository. Vervolgens biedt je deze wijzigingen aan door een "pull request" te doen, een verzoek aan de beheerder om de wijziging toe te voegen aan de VNG Realisatie repository.
+
+De algemene workflow is:
+1. Maak een issue aan voor de gewenste wijziging of toevoeging.
+2. Anderen kunnen op het issue reageren.
+3. Wanneer uit het issue blijkt dat men positief reageert op de voorgestelde wijziging, en je wilt deze gaan realiseren, geef je dat aan in het issue. Wijs het issue toe aan jezelf (hiervoor moet je lid zijn van de community van de repository, je kan verzoeken je toe te laten voegen via de beheerder van de repository).
+4. Maak (wanneer je dat nog niet hebt) een fork van de repository.
+5. Maak in je fork van de repository een branch aan voor de wijziging. Geef deze branch een naam waaran de wijziging te herkennen is.
+6. Voor de wijzigingen en/of toevoegingen door op deze branch in je fork van de repository.
+7. Controleer op basis van criteria (kwaliteitseisen) op de standaard of de bijdrage goed en compleet is.
+8. Maak een pull request.
+9. De beheerder kan ervoor kiezen specifieke leden van de community aan te wijzen om een review te doen op de pull request.
+10. Leden van de community (iedereen die zich belanghebbende vindt of voelt) beoordelen de pull request.
+10. Wanneer in het commentaar op de pull request (noodzakelijke) verbeteringen worden gevraagd, voer je die door in de betreffende branch op je repository.
+11. Wanneer uit de reacties en reviews blijkt dat de pull request goed is, merged te beheerder van de repository het pull request met release-branch de VNG Realisatie repository. (wanneer gewerkt wordt aan de eerste versie van een standaard kan er ook voor worden gekozen direct op de master te mergen).
+
+# Release workflow voor Standaarden
+Standaarden volgen criteria voor versiebeheer en versienummering die voor de (soort) standaard is bepaald.
+Voor het ontwikkelen van een eerste versie (v1.0) van een standaard kan direct op de master worden ontwikkeld. Zodra er een eerste gepubliceerde versie is van de standaard, worden nieuwe versies of patches ontwikkeld vanuit branches voor de versie.
```

Comment:
(Ik begrijp wat je bedoelt maar het is nog wat cryptisch.)
Het komt er volgens mij op neer dat een gepubliceerde versie gepatched kan worden in een eigen branch.
In de master/trunk vindt de doorontwikkeling van de standaard plaats.